canonical-web-and-design/ubuntu.com
# Add /gov page and contact form

## Done

- Add /gov page
- Add /gov contact form

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/gov
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check that the page matches [the copy doc](https://docs.google.com/document/d/1umM1XMEzqeXklPkyCjDr98Q-RvwCcNNXgwNwfLoynTo/edit#)
- Check that the form matches [the copy doc](https://docs.google.com/document/d/1B-2oVj3NU8PYvPWohVYKrYkNFCOUlHS3b31c4J5l_Gk/edit#heading=h.tk4y6g6el1tm) and works as expected


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2847, https://github.com/canonical-web-and-design/web-squad/issues/2848
